app/controllers/concerns/traceable.rb

```diff
@@ -42,10 +42,10 @@ def set_trace_tags
     Rails.logger.error('Error setting service tag', class: self.class.name, message: e.message)
   end
 
-  # Wraps controller methods with the controller name.
+  # Wraps controller methods with the controller name as "origin".
   def tag_with_controller_name(&)
     return yield if controller_name.blank?
 
-    SemanticLogger.named_tagged(controller_name:, &)
+    SemanticLogger.named_tagged(origin: controller_name, &)
```

Comment:
This may not be necessary but` controller_name` removes the `Controller` suffix (`UsersController`-> `users`). Should we use `class.name` so we get `users_controller` and stay consistent with the sidekiq `origin` tag?